src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/domain/events/BookingEventGenerator.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.domain.events
+
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.BookingEntity
+
+class BookingEventGenerator {
+  fun generate(eventName: String, booking: BookingEntity): String {
+    val message = "Generating a $eventName domain event for booking ${booking.id}"
+    println(message)
```

Comment:
Generally try to avoid using `println` as it goes straight to stdout - so miss out on logging levels, log shipping etc.:

```
class BookingEventGenerator {
  private val log = LoggerFactory.getLogger(this::class.java)

  fun generate(eventName: String, booking: BookingEntity): String {
    val message = "Generating a $eventName domain event for booking ${booking.id}"
    log.info("Generating a $eventName domain event for booking ${booking.id}")
    return message
  }
}
```

Reply:
👍